systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -140,6 +143,9 @@ void testJSONFormatLogging(ExtensionContext extensionContext) {
         String configMapKafkaName = "json-layout-kafka";
         String configMapCOName = Constants.STRIMZI_DEPLOYMENT_NAME;
 
+        String originalCoLoggers = kubeClient().getClient().configMaps()
+            .inNamespace(INFRA_NAMESPACE).withName(configMapCOName).get().getData().get("log4j2.properties");
```

Comment:
```suggestion
.inNamespace(clusterOperator.getDeploymentNamespace()).withName(configMapCOName).get().getData().get("log4j2.properties");
```
This needs to change because when you try OLM here (it won't work)

Reply:
Completely forgot that! Good catch 🪲

Reply:
I also changed it everywhere else in test